test/e2e_node/runtime_conformance_test.go

```diff
@@ -318,7 +318,7 @@ while true; do sleep 1; done
 				},
 			} {
 				testCase := testCase
-				It(testCase.description+" [Conformance][NodeConformance]", func() {
+				It(testCase.description+" [NodeConformance]", func() {
```

Comment:
We'll want to test image pulling and image pull secrets (but not host-level secret configuration), but will need to make the registry pulled from configurable.